source/plot/SvgPlot.ooc

```diff
@@ -105,7 +105,7 @@ SvgPlot: class {
 	}
 	setColor: func {
 		if (!datasets empty) {
-			noneColor := ColorRgba new(0, 0, 0, 0)
+			noneColor := ColorRgba black
```

Comment:
Well, this *is* the plotter we have and will have for the forseeable future. Why was this value changed at all?